src/Database/LSMTree/Internal.hs

```diff
@@ -844,7 +846,7 @@ newCursor th = withOpenTable th $ \thEnv -> do
           allocTableContent reg (tableContent thEnv)
         cursorReaders <-
           allocateMaybeTemp reg
-            (Readers.new hfs hbio (Just writeBuffer) (V.toList cursorRuns))
+            (Readers.new hfs hbio offset (Just writeBuffer) (V.toList cursorRuns))
```

Comment:
TODO: it'd probably be better if Readers.new took a vector of runs rather than requiring it be converted to a list.

We don't need to change that in this PR, but lets add a TODO on the definition of `RunReaders.new` before we forget.